docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1505,23 +1508,230 @@ To properly configure the FHIR server's keystore and truststore files, perform t
+the signatureAlgorithm.
+
+Note: the current private key in the fhirKeystore.jks keystore is NOT usable as a signing key.
+Instead, allow Liberty to generate its own keystore and use that.
+
+Assuming the defaultKeyStore has a valid key for signing, add the openidConnectProvider element to your server.xml:
+```
+<openidConnectProvider id="oidc-provider"
+    oauthProviderRef="oauth2-provider"
+    keyStoreRef="defaultKeyStore"
+    signatureAlgorithm="RS256" />
+```
+
+### 5.3.1.3 Request an access token
+After you've registered a client (either via dynamic registration or static config) and configured the openidConnectProvider, invoke
+the token endpoint with the configured code for your client to obtain an access token.
+
+The specific endpoint is determined by the id value of the openidConnectProvider element from the previous section.
+For example, for an OpenID Connect provider with `id="oidc-provider"`, the auth URL will be
+`https://[host]:[port]/oauth2/endpoint/oidc-provider/authorize` and the token URL will be `https://[host]:[port]/oauth2/endpoint/oidc-provider/token`.
+
+### 5.3.2 Configure Liberty as the OpenID Connect Relying Party
+Liberty can be configured to act as an OpenID Connect Relying Party via the [openidConnectClient-1.0 feature](https://openliberty.io/docs/ref/feature/#openidConnectClient-1.0.html).
+
+### 5.3.2.1 Configure the trustStore
+1. Export the server's certificate from the configured keystore via one of the following commands:
+* keytool -exportcert -keystore key.p12 -storepass Password -alias default -file libertyOP.cer
+* keytool -exportcert -keystore key.jks -storepass Password -alias default -file libertyOP.cer
+
+2. Import the certificate into the server's trustStore. Assuming you use the same keystore for both, then use of these:
+* keytool -importcert -keystore key.p12 -storepass Password -alias libertyop -file libertyOP.cer -noprompt
+* keytool -importcert -keystore key.jks -storepass Password -alias libertyop -file libertyOP.cer -noprompt
+
+### 5.3.2.2 Configure server.xml
+Add an openidConnectClient element to the server.xml config and point at the trustStore from the previous section:
+```
+<openidConnectClient id="RS" inboundPropagation="required"
+    clientId="inferno"
+    mapIdentityToRegistryUser="true"
+    trustStoreRef="defaultKeyStore"
+    trustAliasName="libertyop"
+    validationEndpointUrl="https://localhost:9443/oidc/endpoint/oidc-provider/introspect"
+    signatureAlgorithm="RS256"
+    authFilterRef="filter"
+    issuerIdentifier="https://localhost:9443/oauth2/endpoint/oauth2-provider,https://host.docker.internal:9443/oauth2/endpoint/oauth2-provider"
+    />
+
+<authFilter id="filter">
+    <requestUrl urlPattern="/fhir-server" />
+</authFilter>
+```
+
+Note: if the server is also acting as an OAuth/OpenId Connect provider, be sure to include an authFilter to avoid a catch-22 where
+the token and authorization endpoints are themselves protected by the openidConnectClient.
```

Comment:
```suggestion
Note: if the server is also acting as an OAuth/OpenId Connect provider, be sure to include an authFilter to avoid the problem where the token and authorization endpoints are protected by the openidConnectClient.
```